CONTRIBUTING.md

```diff
@@ -102,8 +102,17 @@ Here are some things to keep in mind as you file pull requests to fix bugs, add
 
 ### Release process
 
-- if you aren't sure if a release should happen, open an issue
-- make sure the tests pass
-- `./tools/bump.ts $NEW_VERSION`
-- `node tools/publish.js`
-- update CHANGELOG.md
+- Make sure the tests pass
+- `$ ./tools/bump.ts $NEW_VERSION`
+  - This will commit the changes automatically. Run `git log` to confirm that the changes have been
+    committed.
+  - `$NEW_VERSION` should be an un-prefixed [semantic version](https://semver.org/) number (e.g. `6.0.0-beta.67)
+- `$ node tools/publish.js`
+- Push to default branch
+  - this will create an appropriate tag.
```

Comment:
note that `git push` won't automatically push the tag up. you need to `git push origin master` and `git push origin v1.0.0` separately.

Reply:
Fiddle has a great CONTRIBUTING section for [running a new release](https://github.com/electron/fiddle/blob/main/CONTRIBUTING.md#release-process). Taking some verbiage from there:

```suggestion
- After running the command, you should have a commit updating the version field in the package.json file, as well as a git tag corresponding with the new version number.
- Push your commit upstream to main/default branch.
- Push your tag to the GitHub repository with git push origin <tag_name> .
```
Don't have to use exactly this phrasing, but maybe something like this?